<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAFailProfileSineWgt</class>
 <widget class="QWidget" name="GTAFailProfileSineWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>amplitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amplitudeLE"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="periodLE"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Offset     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="offsetLE"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Phase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phaseLE"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
